.gitignore

```diff
@@ -275,4 +275,19 @@ DerivedData
 *.ipa
 *.xcuserstate
 *.profraw
-sample/ios-app/Marathondevices
\ No newline at end of file
+sample/ios-app/Marathondevices
+
+# Android Sdk
+
+local.properties
+
+## IDEA specific
+.idea/markdown-navigator.xml
+.idea/markdown-navigator-enh.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/runConfigurations.xml
```

Comment:
We do share runConfigurations so that you can import the project and can run a debug session for sample-app. We shouldn't ignore it

Not sure about the other ones